src/main/java/mods/railcraft/common/core/RailcraftConfig.java

```diff
@@ -326,12 +328,7 @@ private static void loadRecipeOption() {
         configMain.addCustomCategoryComment(CAT_RECIPES, "You can add or remove various recipes here");
 
         ConfigCategory cat = configMain.getCategory(CAT_RECIPES);
-        Iterator<String> keys = cat.keySet().iterator();
-        while (keys.hasNext()) {
-            String key = keys.next();
-            if (key.startsWith("recipe"))
-                keys.remove();
-        }
+        cat.keySet().removeIf(key -> key.startsWith("recipe"));
```

Comment:
Check when it was added, its removing old entries. This is probably ancient code.